components/NavbarBootstrap.jsx

```diff
@@ -14,11 +18,30 @@ class NavbarBootstrap extends Component {
             showSignupModal: false,
             showLoginModal: false,
         };
+
+        this.logout = this.logout.bind(this);
+    }
+
+    logout = () => {
+        Router.push({ pathname: '/' });
     }
 
     render() {
         const { showSignupModal, showLoginModal } = this.state;
-        const { isAuthenticated } = this.props;
+        const { isAuthenticated, name } = this.props;
+
+        const popover = (
+            <Popover id="popover-basic" title={name} style={{ textAlign: 'center' }}>
+                <NavDropdown.Item style={{ textAlign: 'center' }}>My Account</NavDropdown.Item>
+                <NavDropdown.Divider />
+                <NavDropdown.Item
+                    style={{ color: 'red', textAlign: 'center' }}
+                    onClick={() => this.logout()}
+                >
+                    Logout
+                </NavDropdown.Item>
```

Comment:
Do we want to add an "Add car" dropdown item here as well? not sure if that was in the finalized design

Reply:
this is just the preliminary design/code - we can add whatever stuff like `add car` at a later time I think :) what do you say? (i dont think it should be a blocker for this PR)

Reply:
Sounds good